Jenkinsfile

```diff
@@ -0,0 +1,96 @@
+// This Jenkinsfile is used by Jenkins to run the 'FireworksLayout' step of Reactome's release.
+// It requires that the 'DiagramConverter' step has been run successfully before it can be run.
+
+import org.reactome.release.jenkins.utilities.Utilities
+
+// Shared library maintained at 'release-jenkins-utils' repository.
+def utils = new Utilities()
+
+pipeline{
+	agent any
+	// Set output folder that will contain files created by step.
+	environment {
+        	OUTPUT_FOLDER = "fireworks"
+    	}
+
+	stages{
+		// This stage checks that upstream project 'DiagramConverter' was run successfully.
+		stage('Check DiagramConverter build succeeded'){
+			steps{
+				script{
+				    utils.checkUpstreamBuildsSucceeded("File-Generation/job/DiagramConverter/")
+				}
+			}
+		}
+		// This stage builds the jar file using maven.
+		stage('Setup: Build jar file'){
+			steps{
+				script{
+					sh "mvn clean package"
```

Comment:
I thought you had a special "build jar" function in the shared library. Why not use it here?

Reply:
These projects use a different build command (`mvn clean package` here vs. `mvn clean compile assembly:single`) - I am going to look into having a unified function for the next iteration

Reply:
Fair enough. If we could get all (or most) of our projects built using the _same_ maven command, that would be even better!